### PR TITLE
[mesheryctl] Add the mesheryctl build as multiple platforms

### LINKED ISSUE #1002: mesheryctl support build for multiple platforms
**Current Behavior**

The `mesheryctl`  Makefile need to upgrade and we should standard to use the `goreleaser` as the build tool.

**Desired Behavior**

The new Makefile for `mesheryctl` can build for multiple platforms soon at local

---
**Resources**

**Alternatives / Additional Context**

### PR DESCRIPTION
**Pull request by Aisuko**
Signed-off-by: Aisuko <urakiny@gmail.com>

#### Description

This PR fixes #1002  

#### Checklist

- [x] [DCO](https://github.com/layer5io/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin) signed.
- [x] Title of the PR starts with area name, e.g. `meshery:`, `mesheryctl:`, `ui:`.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -5,12 +5,43 @@ MESHERY_CLOUD_DEV=http://localhost:9876
 MESHERY_CLOUD_PROD=https://meshery.layer5.io
 MESHERY_CLOUD_STAGING=https://staging-meshery.layer5.io
 
+
```
Comment (leecalcote) [inline Makefile:8]: @Aisuko you may want to replicate a version of these changes to https://github.com/layer5io/meshery/blob/master/mesheryctl/Makefile

**Comment by leecalcote** [comment]
"Backend build" check failed. Re-running jobs...

File: Makefile (hunk 1 above)
Comment (Aisuko) [reply Makefile]: Done

**Comment by Aisuko** [comment]
May we add this into `mesheryctl` folder's Makefile?  Although it can work well maybe we can aggregate `mesheryctl` into one Makefile.

https://github.com/layer5io/meshery/blob/6219b63f2151e8fc953e3c238c0fd20d4954c8b6/Makefile#L10

**Comment by leecalcote** [comment]
> May we add this into `mesheryctl` folder's Makefile? Although it can work well maybe we can aggregate `mesheryctl` into one Makefile
> 
> https://github.com/layer5io/meshery/blob/6219b63f2151e8fc953e3c238c0fd20d4954c8b6/Makefile#L10-L12



> May we add this into `mesheryctl` folder's Makefile? Although it can work well maybe we can aggregate `mesheryctl` into one Makefile
> 
> https://github.com/layer5io/meshery/blob/6219b63f2151e8fc953e3c238c0fd20d4954c8b6/Makefile#L10-L12

Sounds great, @Aisuko

File: mesheryctl/Makefile [hunk 2]
```diff
@@ -1,4 +1,33 @@
-make:
-	go build -o mesheryctl
-forwin:
-	go build -o mesheryctl.exe
\ No newline at end of file
+OUTDIR := _output
```
Comment (kushthedude) [inline mesheryctl/Makefile:1]: Why we have removed the make command from here ?

**Pull request merged**